backend/orchestrator/project.json

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
in `swarmion/template` we decided to only define targets in `nx.json` and not in `project.json` anymore
if we want our backend to be independent from the rest it makes sense, but I find it weird having both conventions

Reply:
This is fixed in a separate commit